rx/concurrency/mainloopscheduler/qtscheduler.py

```diff
@@ -51,39 +69,43 @@ def dispose():
 
         return CompositeDisposable(sad, Disposable(dispose))
 
-    def schedule_absolute(self, duetime: typing.AbsoluteTime, action: typing.ScheduledAction,
-                          state: typing.TState = None) -> typing.Disposable:
+    def schedule_absolute(self,
+                          duetime: typing.AbsoluteTime,
+                          action: typing.ScheduledAction,
+                          state: Optional[typing.TState] = None
+                          ) -> typing.Disposable:
         """Schedules an action to be executed at duetime.
 
         Args:
             duetime: Absolute time after which to execute the action.
             action: Action to be executed.
+            state: [Optional] state to be given to the action function.
 
         Returns:
             The disposable object used to cancel the scheduled action
             (best effort).
         """
 
         duetime = self.to_datetime(duetime) - self.now
-        return self.schedule_relative(duetime, action, state)
+        return self.schedule_relative(duetime, action, state=state)
 
-    def schedule_periodic(self, period: typing.RelativeTime, action: typing.ScheduledPeriodicAction,
-                          state: typing.TState = None):
-        """Schedules a periodic piece of work to be executed in the Qt
-        mainloop.
+    def schedule_periodic(self,
+                          period: typing.RelativeTime,
+                          action: typing.ScheduledPeriodicAction,
+                          state: Optional[typing.TState] = None
+                          ) -> typing.Disposable:
+        """Schedules a periodic piece of work to be executed in the loop.
 
-        Args:
-            period: Period in milliseconds for running the work
-                periodically.
+       Args:
+            period: Period in seconds for running the work repeatedly.
             action: Action to be executed.
-            state: [Optional] Initial state passed to the action upon
-                the first iteration.
+            state: [Optional] state to be given to the action function.
 
         Returns:
-            The disposable object used to cancel the scheduled
-            recurring action (best effort).
+            The disposable object used to cancel the scheduled action
+            (best effort).
         """
-        msecs = int(self.to_seconds(period)*1000.0)
+        msecs = int(self.to_seconds(period)) * 1000
```

Comment:
Should it be `msecs = int(self.to_seconds(period) * 1000)` instead? 
E.g. I feel that 1.6s will be rounded to 1s resulting in 1000ms instead of 1600ms.

Reply:
Ah, thanks! I accidentally deleted a parenthesis here and restored it in the wrong place... Will amend immediately.